src/SystemTemplates.cc

```diff
@@ -782,6 +932,25 @@ static const unordered_map<uint32_t, ResourceFile::TemplateEntryList> system_tem
   {RESOURCE_TYPE_PSAP, {
     t_pstring("String"),
   }},
+  {RESOURCE_TYPE_pslt, {
+    t_word("Number of Nubus pseudo-slots"),
+    // 0 = Horizontal form factor, ascending slot order   
+    // 1 = Horizontal form factor, descending slot order  
+    // 2 = Vertical form factor, ascending slot order     
+    // 3 = Vertical form factor, descending slot order

```

Comment:
Should these go in case_names for the below field?

Reply:
They should, of course. Fixed.